TeaDiary/Scripts/App/displayTea.js

```diff
@@ -0,0 +1,32 @@
+﻿function TeaDisplayer(containers) {
+    var self = this;
+    self.containers = containers;
+    var name = $(self.containers.Name);
+    var secondName = $(self.containers.SecondName);
+    var type = $(self.containers.Type);
+    var notes = $(self.containers.Notes);
+    function clearContainers() {
+        name.html('');
+        secondName.html('');
+        type.html('');
+        notes.html('');
+    }
+    
+    self.displayTea = (tea) => {
+        clearContainers();
+        name.val(tea.Name);
```

Comment:
Here you have two `name`s. One is a value. The other is a reference to an HTML element. [It's confusing](https://www.youtube.com/watch?v=IjOkC-YYNJo).

A better way is to have `name` and `nameElement` or `nameEditor`